client/src/components/base/ItemLists/ItemLabel/ItemLabel.tsx

```diff
@@ -0,0 +1,67 @@
+import styled from '@emotion/styled';
+import React from 'react';
+
+import { baemin, green, red1 } from '@/static/style/common';
+
+const feature = ['NEW', 'BEST', 'SALE', 'GREEN'];
+
+type LabelProps = {
+  quantity: number;
+};
+
+const ItemLabel = ({ quantity }: LabelProps) => {
+  const getColor = (feature) => {
+    switch (feature) {
+      case 'NEW':
+        return newFeatureColor;
+
+      case 'BEST':
+        return bestFeatureColor;
+
+      case 'SALE':
+        return saleFeatureColor;
+
+      case 'GREEN':
+        return greenFeatureColor;
+    }
+  };
+
+  const createFeature = () => {
+    return feature.map((item, idx) => (
+      <Feature color={getColor(item)} key={idx}>
+        <span>{item}</span>
+      </Feature>
+    ));
+  };
+
+  return <FeatureContainer>{createFeature()}</FeatureContainer>;
+};
```

Comment:
```suggestion
  const Features = () => (
      <>
        {feature.map((item, idx) => (
          <Feature color={getColor(item)} key={idx}>
            <span>{item}</span>
          </Feature>
        ))}
      </>
    );

  return <FeatureContainer><Features /></FeatureContainer>;
};
```
좋은 방법인지는 잘 모르겠지만 이렇게 하면 좀 더 보기 좋다고 해야할까... ㅋㅋ 무튼 참고하세요~

Reply:
오 그러네요 더 보기 좋아요!! 감사합니다 👍